Tests/DistributedActorsTests/TraversalTests.swift

```diff
@@ -86,6 +86,7 @@ final class TraversalTests: ClusterSystemXCTestCase {
                 // "downingStrategy", "swim", "nodeDeathWatcher", "gossip", "cluster", "leadership",
                 // end of cluster actors ---
                 "clusterEvents",
+                "clusterMembership",
```

Comment:
I think this one is fine as not-distributed actor; so it won't be in the tree either.